dockerfiles/piraeus-server/Dockerfile

```diff
@@ -31,5 +31,10 @@ EXPOSE 3366/tcp 3367/tcp
 
 COPY entry.sh /usr/bin/piraeus-entry.sh
 
+ARG K8S_AWAIT_ELECTION_ARCH
+ARG K8S_AWAIT_ELECTION_VERSION
+
+RUN wget https://github.com/LINBIT/k8s-await-election/releases/download/v0.1.0/k8s-await-election-${K8S_AWAIT_ELECTION_VERSION}-linux-${K8S_AWAIT_ELECTION_ARCH}.tar.gz -O - | tar -xvz -C /usr/bin/
```

Comment:
`...download/v0.1.0/k8s-awai...`. Guess that static part should also be replaced with `${K8S_AWAIT_ELECTION_VERSION}`